src/StoreApi/Schemas/V1/OrderSchema.php

```diff
@@ -271,8 +271,8 @@ public function get_item_response( $order ) {
 			'coupons'              => $this->get_item_responses_from_schema( $this->coupon_schema, $order->get_items( 'coupon' ) ),
 			'fees'                 => $this->get_item_responses_from_schema( $this->fee_schema, $order->get_items( 'fee' ) ),
 			'totals'               => (object) $this->prepare_currency_response( $this->get_totals( $order ) ),
-			'shipping_address'     => (object) $this->shipping_address_schema->get_item_response( new \WC_Customer( $order->get_customer_id() ) ),
-			'billing_address'      => (object) $this->billing_address_schema->get_item_response( new \WC_Customer( $order->get_customer_id() ) ),
+			'shipping_address'     => (object) $this->shipping_address_schema->get_item_response( $order ),
```

Comment:
This is because an order does not necessarily have a customer. This will also get guest details.